src/inlineExpression.tsx

```diff
@@ -91,7 +91,7 @@ export class RenderedExpression extends Widget {
 }
 
 function PlainTextRenderer({ content }: { content: string }) {
-  content = content.replace(/^(["'])(.*)\1$/, '$2');
+  //content = content.replace(/^(["'])(.*)\1$/, '$2');
```

Comment:
Just browsing the most recent changes, this was in to remove quotes around strings. Is this change intentional?